setup.py

```diff
@@ -0,0 +1,45 @@
+import os
+import re
+from setuptools import setup, find_packages
+from setuptools.command.install_scripts import install_scripts
+
+
+class my_install_scripts(install_scripts):
+    def write_script(self, script_name, contents, mode="t", *ignored):
+        contents = re.sub("import sys",
+                          "import sys\nsys.path.append('/opt/graphite/lib')",
+                          contents)
+        install_scripts.write_script(
+            self, script_name, contents, mode="t", *ignored)
+
+
+def read(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+setup(
+    name="carbonate-utils",
+    version="0.1",
+    author="Corentin Chary",
```

Comment:
there is a maintainer field, either should be sufficient (though I appreciate the README ;) )